modules/ROOT/pages/basketball-robot-guide/software/review-summary.adoc

```diff
@@ -1,12 +1,40 @@
 :toc:
 
-= Code review summary
+== Requirements for code and reviews
 
-== Follow style guide for Python Code (PEP 8), if the team does that have any other preferences
+=== Created Merge Requests should be small in scope
+
+To make Merge Requests as smooth as possible let's keep one MR-s scope to one task.
+
+If multiple tasks are combined into one MR it becomes hard to review due to the sheer scope of changes. This increases the time a review takes. Additionally, it is harder for the codes author to fix issues since systematic mistakes can be repeated more, creating more places needing fixing.
```

Comment:
Write single sentence per line and split in reasonable place when the line is long.

https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line